libs/utils/utils.js

```diff
@@ -389,6 +389,7 @@ function checkForExpBlock(name, expBlocks) {
 }
 
 export async function loadBlock(block) {
+  if (block.classList.contains('hide-block')) return block;
```

Comment:
Does it even make sense to keep it? Could it not be removed entirely? DOM would be a bit cleaner, there wouldn't be a need for the CSS class, it would avoid issues if someone decides that `.quote.center` needs a `display:flex !important` for whatever reason. Just my 2 cents